R/get_boxi_extract_path.R

```diff
@@ -83,3 +59,46 @@ get_dummy_boxi_extract_path <- function() {
     create = TRUE
   )
 }
+
+#' Get path as a csv or csv.gz
+#'
+#' @param year Year of extract
+#' @param type Name of BOXI extract passed from [get_boxi_extract_path]
+#' @param ext Choice of extension between csv or csv.gz for zipped extracts
+#'
+#' @return The file path to boxi extracts. Note for DN and CMH data this is
+#'         now in an archived folder which is set up to point to this in
+#'         the selections.
+#' @export
+#'
+get_csv_gz_path <- function(year, type, ext = c("csv", "csv.gz")) {
+  if (type %in% c("DN", "CMH")) {
+    dir <- fs::path(get_slf_dir(), "Archived_data")
+  } else {
+    dir <- get_year_dir(year, extracts_dir = TRUE)
+  }
```

Comment:
I'm not sure what the benefit of a new function is. What would be the downside of just including this snippet in `get_boxi_extract_path`, in place of the current `year_dir <- ...` bit?

Reply:
Thanks @Moohan I think i had over complicated this and just pushed the changes. I had tried that but then the function was not working properly for me. Possibly due to having too many things open at the time. I could try this again and remove the new function